### ISSUE TITLE
[Webform] Associated checkboxes and radio buttons do not announce the group heading when navigating with screen-reader (Level A fail)

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#877]
As a user with a screen-reader, I need to be able to hear the group heading for related checkboxes or radio buttons.

In the example below, when navigating in screen-reader Focus Mode, I should be able to tell that the checkboxes are grouped under the heading "Question Category".

![image](https://user-images.githubusercontent.com/1927902/47571303-365e9700-d906-11e8-9c2a-b473715f66aa.png)

This fails 1.3.1 Info and Relationships (Level A) as well as 3.3.2 Labels or Instructions (Level A)
